### PR TITLE
heh: init at 0.4.1

### PR DESCRIPTION
**Pull request by Piturnah**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/heh/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-IIF/bkTLwR8pCs/hJ625T3NsiKf/6Zf1cW2i4lsiK4U=";
+  };
+
+  cargoHash = "sha256-tDvqaNVuzv1BlS/oNI1D/WV1b5uHreT3Ak/6ruqKXQc=";
+
+  meta = with lib; {
+    description = "A cross-platform terminal UI used for modifying file data in hex or ASCII.";
+    homepage = "https://github.com/${src.owner}/${src.repo}";
```
Comment (davidak) [inline pkgs/applications/editors/heh/default.nix:22]: Please use the raw URL, so it can easily be opened. The `rev` is fine for the `changelog`.

**Comment by davidak** [comment]
Result of `nixpkgs-review pr 246402` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>heh</li>
  </ul>
</details>

**Review by davidak — CHANGES_REQUESTED** [review]
Code looks good

Builds, works

Please change the one nitpick. Then it's good to merge from my side.

File: pkgs/applications/editors/heh/default.nix (hunk 1 above)
Comment (Piturnah) [reply pkgs/applications/editors/heh/default.nix]: Fixed :+1:

**Comment by davidak** [comment]
Thanks. Can you squash the commits into one?

**Comment by davidak** [comment]
Result of `nixpkgs-review pr 246402` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>heh</li>
  </ul>
</details>

**Review by davidak — APPROVED** [review]
Code looks good

Builds, works

Thanks for your contribution! :sparkles:

**Pull request merged**